.github/workflows/automerge-dependabot.yml

```diff
@@ -0,0 +1,28 @@
+---
```

Comment:
not sure if forks will have this, maybe we should disable it for them

Reply:
done!